subjects/firstword/README.md

```diff
@@ -2,22 +2,38 @@
 
 ### Instructions
 
-Write a program that takes an argument and displays its first word, followed by a newline (`'\n'`).
+Write a function that takes a string and return a string containing its first word, followed by a newline (`'\n'`).
 
 - A word is a sequence of characters delimited by spaces or by the start/end of the argument.
 
-- The output will be followed by a newline (`'\n'`).
+### Expected Function
 
-- If the number of arguments is not 1, or if there are no words, the program displays nothing.
+```go
+func FirstWord(s string) string {
+    // ...
+}
+```
 
 ### Usage
 
+Here is a possible way to test your function:
+
+```go
+package main
+
+import "fmt"
+
+fmt.Print(FirstWord("hello there"))
```

Comment:
missing `func main(){}` and the package piscine as it is in the lasword exercise

Reply:
Thank you for the catch :pray: